server/src/controllers/GlimpseController.js

```diff
@@ -0,0 +1,10 @@
+require("dotenv").config();
+const axios = require("axios");
+const extractPhotos = require("../utility/extractPhotos");
+
+module.exports = {
+  async getPhotos(_req, res) {
+    const response = await axios.get(process.env.GPHOTOSURL);
```

Comment:
This takes like 2-3 seconds to get the data, once this gets done, #34 can be updated to include this as well